### DISCUSSION TITLE
Show original official scenario names from WW and TT in the side bar

### DISCUSSION THREAD

**Discussion by MASHzj** [OpenRCT2/OpenRCT2#22070] [Feature requests]
After a long discussion  #21214 has now been approved. I'm not happy with it, but more people voted to show the park names in the main column, so I'm ok with that. However, the poll also showed a significant amount of people against it. To cater to them, the original/official could be shown in the side bar where the park description and goal are (where exactly doesn't really matter to me). Some including me already asked that in the PR's discussion thread, but after merge it's better to ask it here again.

**Comment by HtotheTML** [comment]
That's a really good idea; I don't want the old names to completely disappear into oblivion. Having the original name display in the sidebar description would be a happy medium.

**Reply by MASHzj** [reply]
Thanks for your understanding 😉

**Comment by MarcelVos96** [comment]
I agree with this, although personally I'd prefer something even more visible. Putting them in the sidebar is something, but doing something like "Sherwood Forest (Robin Hood)" is an even better solution in my opinion.

This might be a bit of an issue with long scenario names becoming really wide, but there's probably a solution for that.

**Comment by Gymnasiast** [comment]
I suggest that everyone who is apprehensive just gives it a chance, rather than asking for watering it down (or reversing) straight away. We can revisit this in six months or so.

**Reply by MASHzj** [reply]
Fine for me to wait some months. But I don't understand what my proposal would water down (and I certainly don't ask to revert it). The reason I started this discussion is because in the PR discussion thread, I expressed my concerns and also did suggestions for alternative approaches, but ultimately the change was just pushed through. I leave in the middle if that was intentional, but I didn't feel taken serious at all. And now that the PR is closed, those comments (also from other opponents) are not very visible anymore. And for me it is about more than just getting used to something new.
